src/arkode/arkode_arkstep.c

```diff
@@ -1341,7 +1319,7 @@ int arkStep_FullRHS(void* arkode_mem, realtype t, N_Vector y, N_Vector f,
         Xvecs[0] = step_mem->Fe[0];
         nvec     = 1;
         arkStep_ApplyForcing(step_mem, t, ONE, &nvec);
-        N_VLinearCombination(nvec, cvals, Xvecs, step_mem->Fe[0]);
+        SUNCheckCallNoRet(N_VLinearCombination(nvec, cvals, Xvecs, step_mem->Fe[0]));
```

Comment:
Since ARKODE should fail if this linear combination is unsuccessful, then we should catch an error and return (unlike my previous implementation that forgot to check the output of `N_VLinearCombination`).

```suggestion
        SUNCheckCall(N_VLinearCombination(nvec, cvals, Xvecs, step_mem->Fe[0]));
```

Reply:
This goes back to the return type of the integrator function changing.